test/Tools/NodeDocumentationMarkdownGeneratorTests/MarkdownGeneratorTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NodeDocumentationMarkdownGenerator;
+using NUnit.Framework;
+
+namespace NodeDocumentationMarkdownGeneratorTests
+{
+    public class MarkdownGeneratorTests
+    {
+        private static string coreDirectory = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
in addition to tests of this CLI tool - we will need to add tests to many of the new classes and methods that were added to DynamoCore or our coverage will go down.

They can be very simple units tests of the functions though.